generator/generate_md.py

```diff
@@ -0,0 +1,51 @@
+class Task:
```

Comment:
Prefix must be one of the following ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR'].